Makefile

```diff
@@ -152,8 +152,8 @@ nessie: ## run nessie (system testing)
 
 test: test-go test-hadoopfs  ## Run tests for the project
 
-test-go: gen
-	$(GOTEST) -count=1 -coverprofile=cover.out -race -cover -failfast $(GO_TEST_MODULES)
+test-go: gen			# Run parallism 4: most of our slow tests are *not* CPU-bound.
+	$(GOTEST) -count=1 -coverprofile=cover.out -race -cover -failfast --parallel 4 $(GO_TEST_MODULES)
```

Comment:
can we control this one outside the makefile so in case you run it locally it will use all of our cores?

Reply:
Yes, but I don't want to.  Our slow tests are not CPU-bound.  There is little to no point in trying to run 1 test on every core.  (Actually there is little to no point in trying to run 1 test on every core even for CPU-bound tests, because some tests are multithreaded!)
If we could attach a "weight" or a set of measures to each test, then it would be possible to schedule them based on the size of the machine.  As it stands, it probably doesn't make much difference what we do here :-/

Reply:
can you at least set the value based on make variable we can override it

Reply:
Done!